.github/workflows/rust.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: taiki-e/install-action@nextest
 
       - name: SDK tests
-        run: cargo nextest run --all
+        run: cargo nextest run --no-tests=warn --all
```

Comment:
Because `nextest` was updated, and will throw an error if it finds no tests. So we either remove the step altogether, or do this. I can remove the step completely if you prefer that :-)